printing/prettyp.ml

```diff
@@ -730,9 +730,9 @@ let print_full_pure_context env sigma =
                 print_basename con ++ str " : " ++ cut () ++ pr_ltype_env env sigma typ
 	      | OpaqueDef lc ->
 		str "Theorem " ++ print_basename con ++ cut () ++
-                str " : " ++ pr_ltype_env env sigma typ ++ str "." ++ fnl () ++
-                str "Proof " ++ pr_lconstr_env env sigma (fst (Opaqueproof.force_proof Library.indirect_accessor (Global.opaque_tables ()) lc))
-	      | Def c ->
+                str " : " ++ pr_ltype_env env sigma typ ++ str "." ++ fnl ()
+                (* str "Proof " ++ pr_lconstr_env env sigma (fst (Opaqueproof.force_proof Library.indirect_accessor (Global.opaque_tables ()) lc)) *)
```

Comment:
Why was this commented out?

Reply:
Indeed, a left over from the previous version, fixed.